packages/phone-number-privacy/src/salt-generation/query-quota.ts

```diff
@@ -0,0 +1,56 @@
+import { AttestationsWrapper } from '@celo/contractkit/lib/wrappers/Attestations'
+import fetch from 'node-fetch'
+import config from '../config'
+import { getPerformedQueryCount } from '../database/wrappers/account'
+import { getContractKit } from '../web3/contracts'
+
+const NUM_ATTESTATIONS_REQUIRED = 3
```

Comment:
ditto, move to config and don't duplicate it

Reply:
oh it was unused in the other location... but i'll put in config anyway